docker_checker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM rocker/r-devel
+MAINTAINER karl.forner@quartzbio.com
+
+RUN Rscript -e 'install.packages("digest")'
```

Comment:
Is there a reason you can't install all the packages in one statement?  Also you should only need to install the packages in Imports 
```
    jsonlite,
    rex,
    devtools,
    httr,
    testthat,
    crayon
```